crates/sui-benchmark/tests/simtest.rs

```diff
@@ -160,6 +178,21 @@ mod test {
                 handle_failpoint(dead_validator.clone(), client_node, 0.01);
             }
         });
+
+        // Narwhal fail points.
+        let dead_validator = dead_validator_orig.clone();
+        register_fail_points(
+            &[
+                "narwhal-rpc-response",
+                "narwhal-store-before-write",
+                "narwhal-store-after-write",
```

Comment:
where do we define the behavior for these?

Reply:
Behavior is defined by the associated handler function that is passed to the registration function